<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeEditorControls</class>
 <widget class="QWidget" name="NodeEditorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>996</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>4</y>
     <width>471</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Moving Node</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkSingleNodeSelectionWidget" name="widget_MovingNode" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_applyLandMark">
        <property name="text">
         <string>Apply LandMark</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_applyIcp">
        <property name="text">
         <string>Apply ICP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_undo">
        <property name="text">
         <string>Un do</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Source LandMarks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkSingleNodeSelectionWidget" name="widget_SourcePset" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Target LandMarks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkSingleNodeSelectionWidget" name="widget_TargetPset" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Surface Registration</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTextBrowser" name="textBrowser_trans_info"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_47">
        <property name="text">
         <string>ICP Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkSingleNodeSelectionWidget" name="widget_IcpPset" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
